agents/lakeshore240/LS240_agent.py

```diff
@@ -202,21 +220,15 @@ def start_acq(self, session, params=None):
                     'data': {}
                 }
 
-                if self.fake_data:
-                    for therm in self.thermometers:
-                        data['data'][therm + ' T'] = random.randrange(250, 350)
-                        data['data'][therm + ' V'] = random.randrange(250, 350)
-                    time.sleep(.2)
-
-                else:
-                    for i, therm in enumerate(self.thermometers):
-                        data['data'][therm + ' T'] = self.module.channels[i].get_reading(unit='K')
-                        data['data'][therm + ' V'] = self.module.channels[i].get_reading(unit='S')
-
-                    time.sleep(sleep_time)
+                for chan in self.module.channels:
+                    chan_string = "channel_{}".format(chan.channel_num)
```

Comment:
I'll restate my previous comment before the recent changes. I think moving to "channel_" from "Channel " is going to cause thermometers that are already in use to have a different name saved within the g3 files (differing in the capitalization of Channel and the use of a '_' instead of a ' ') which might cause some users issues. I think we should stick with the "Channel " formatting until we implement the channel map we've discussed.